bridge/tests/db/test_WrapRequestDatabase.py

```diff
@@ -902,11 +922,36 @@ def test_can_lookup_block_height_when_present(self):
 
 	# endregion
 
+	# region set_payout_block_metadata
+
+	def test_can_set_payout_block_timestamp(self):
+		# Arrange:
+		def post_insert_action(database):
+			database.set_payout_block_timestamp(1122, 98765)
+
+		# Act + Assert:
+		self._assert_can_insert_requests([], [], expected_payout_block_metadatas=[
+			(1122, self._symbol_to_unix_timestamp(98765))
+		], post_insert_action=post_insert_action)
+
+	def test_can_overrwrite_payout_block_timestamp(self):
```

Comment:
In what case will the payout block be overwritten?  If the payout transaction fails? 🤔

Reply:
i don't think this _should_ happen in current workflows, but just wanted to have test to define behavior in case that changes